subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,66 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more and more popular. Are you wondering why? Because a
+subscription model simplifies the business process and makes it very easy for both customers and the
+business. It ensures that the business retains more customers on a consistent basis and that clients
+can be reached out to and engaged with regularly. A business model like this makes it easy to
+anticipate demand and supply. Finally, this model offers much higher payment security for your
+business.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
+   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
+   management.</b> </div>
+
+Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
+help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
+possibility to create, edit and manage your own subscription templates.
+
+Configuration
+=============
+
+Go to :menuselection:`Subscriptions → Configuration → Subscription templates`. By default, Odoo
+suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription).
+Of course, you can create your own ones.
+
+.. image:: media/subscription_templates_1.png
+  :align: center
+  :alt: How to use subscription templates on Odoo Subscriptions?
+
+.. important::
+   Installing the **Subscriptions** application will automatically install the **Sales** and
+   **Invoicing** applications.
+
+Create your first template
+==========================
+
+You can create a new template or edit an existing one. The first thing you need to do is give your
+template a name. After that, you have to choose an invoicing period. It is possible to change the
+amount and specify whether it is per days, weeks, months or years. The duration will determine if
+the subscription will go on forever (until it’s manually closed) or for a fixed amount of time.
+There are also several options for the payment mode. Something important here is that an additional
+field appears when you choose “Send”, “Send & try to charge” and “Send after successful payment”.
+This field gives you the possibility to add an invoice email template (as you can see below).
+
+.. image:: media/subscription_templates_2.png
+  :align: center
+  :alt: How to use subscription templates on Odoo Subscriptions?
+
+On each template, you can also configure some options for the customer portal. First of all, you are
+going to be able to say if you want the customer to be able to close the subscription or not. After
+that, you have the automatic closing limit, the group of subscription and the journal options that
+you can also manage directly from the subscription templates.
```

Comment:
```suggestion
For each template, you can also choose if you want the customer to be able to close their subscription or not. If enabled (I'm guessing this hehe), you can set an *Automatic Closing* limit, the *Group of Subscription* and the *Journal* options. You can also manage these options directly from the subscription templates.
```

I feel like these features 'automatic, group, journal' could have a little explanation? no? I believe that this is the reason we mention features in text, to explain them, at least most of the time.

Reply:
It is a link with the Accounting app. If I give an explanation for the three of them, it will be too long I guess. Corentin, the ex PO of subscriptions, told me that it is not necessary to explain these fields :/

Reply:
ohh ok!